### PR TITLE
Fixed stutter due to very large tooltip text

### LINKED ISSUE #444: Single Click Doesn't work on Long names in Chart Summary View
### Description
Refer to the video below, I observed when the filter name is too long, single click that generally opens the Table View doesn't work. On clicking twice or thrice, the Table View opens up.

Single-Click-Doesn_t-Work

### PR DESCRIPTION
**Pull request by OmkarPh**
Fixes #444 

Signed-off-by: Omkar Phansopkar <omkarphansopkar@gmail.com>

### CODE REVIEW

**Comment by OmkarPh** [comment]
Trimmed the tooltip text, If you want, we can change the trim character count
image

**Comment by OmkarPh** [comment]
@pombredanne ^^

**Comment by AyanSinhaMahapatra** [comment]
LGTM! Thanks @OmkarPh!

Tested and can confirm that the stutter is not present anymore after this fix is applied.

File: assets/app/js/helpers/barChart.js [hunk 1]
```diff
@@ -112,7 +112,7 @@ class BarChart {
         tooltip
```
Comment (AyanSinhaMahapatra) [inline assets/app/js/helpers/barChart.js:112]: There's another `.text()` on line 96, is that something (and are there other places like this elsewhere in charts/others) that would also benefit from trimming potentially?

I'm not super familiar with .js or this codebase so I have no idea btw.

Comment (OmkarPh) [reply assets/app/js/helpers/barChart.js:112]: That line is used to set the count for the bar which won't be that long character wise

to verify i added console log on that line to check what values are used there i.e.
`      .text((d) => { console.log('text being set on line 96 to', d.val); return '(' + d.val + ')'; })
`
It gave output:
image


and, about trimming at other places, I searched through a lot of files but only these 2 needed change

**Review by steven-esser — APPROVED** [review]
This looks good.

**Pull request closed without merging**